### PR TITLE
[IMP] Website_sale: shop grid layout

### PR DESCRIPTION
**Pull request by qha-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: t [hunk 1]
```diff
@@ -0,0 +1,56 @@
+[1mdiff --cc addons/website_sale/static/src/js/website_sale.editor.js[m
```
Comment (seb-odoo) [inline t]: This file can be removed

**Comment by qsm-odoo** [comment]
@qha-odoo Could you rebase this? Master has changed quite a lot. Thanks.

**Comment by qha-odoo** [comment]
@qsm-odoo It's done

**Comment by qsm-odoo** [comment]
Continuing on https://github.com/odoo/odoo/pull/31147

**Pull request closed without merging**